DEVELOP.rst

```diff
@@ -88,11 +88,17 @@ To compile run::
 
     ./gradlew compileJava
 
+Before running, a config file for log4j is required. Create an empty default with::
+
+    touch config
+
+(For individual configuration see http://logging.apache.org/log4j/1.2/faq.html#noconfig)
```

Comment:
can you make a link out of the URL http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#hyperlink-targets
also: are there any useful defaults to show here?